### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by jevazquezb**
In this part of the project we:

- Inserted some more data into the "animals" table.
- Used transactions to update and delete records.
- Applied some aggregate functions and the GROUP BY statement to answer analytical questions (queries).

The results of the transactions and queries are shown below.

- Inside a transaction update the animals' table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
transaction1

- Inside a transaction:
  - Update the animals' table by setting the species column to digimon for all animals that have a name ending in "mon".
  - Update the animals' table by setting the species column to pokemon for all animals that don't have species already set.
  - Commit the transaction.
  - Verify that change was made and persists after commit.
transaction2

  - *New session (changes persist after committing)*:
transaction2_1

- Inside a transaction delete all records in the animals' table, then roll back the transaction. After the rollback verify if all records in the animals' table still exist.
transaction3

- Inside a transaction:
  - Delete all animals born after Jan 1st, 2022.
  - Create a savepoint for the transaction.
  - Update all animals' weight to be their weight multiplied by -1.
  - Rollback to the savepoint.
  - Update all animals' weights that are negative to be their weight multiplied by -1.
  - Commit the transaction.
transaction4
transaction4_1

  - *New session (changes persist after committing)*:
transaction4_2

- Write queries to answer the following questions:
  - How many animals are there?
  - How many animals have never tried to escape?
  - What is the average weight of animals?
  - Who escapes the most, neutered or not neutered animals?
  - What is the minimum and maximum weight of each type of animal?
  - What is the average number of escape attempts per animal type of those born between 1990 and 2000?
queries3

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,3 +4,13 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, TRUE, 8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-07', 1, FALSE, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, TRUE, 11);
+INSERT INTO
+  animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+  ('Charmander', '2020-02-08', 0, FALSE, -11),
+  ('Plantmon', '2021-11-15', 2, TRUE, -5.7),
+  ('Squirtle', '1993-04-02', 3, FALSE, -12.13),
+  ('Angemon', '2005-06-12', 1, TRUE, -45),
```
Comment (zieeco) [inline data.sql]: - Please, kidly add a  `dot` and zero to the numbers as they're floating numbers to meet the project requirement.
`-11.0`, ` -5.7`, `-12.13` and `-45.0`

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @jevazquezb,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice schema setup. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (jevazquezb) [reply data.sql]: @zieeco Thanks for the observation. I already added them. 👍

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @jevazquezb 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**